api/crossref/permissions.py

```diff
@@ -22,6 +25,10 @@ def has_permission(self, request, view):
             ),
             digestmod=hashlib.sha256,
         ).hexdigest()
-        if signature != data.get('signature', None):
-            raise PermissionDenied('Invalid headers on incoming mail')
+        if 'signature' not in data:
+            exc = 'Signature required in request body'
+            sentry.log_exception(exc)
```

Comment:
I believe you want `sentry.log_message` here instead of `sentry.log_exception`.

https://docs.sentry.io/clients/python/api/#raven.Client.captureMessage